charts/shoot-core/charts/csi-alicloud/charts/utils-templates/Chart.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
Is this really a copy? This should be a symlink, please change (see README.md)

Reply:
done. I changed all the hard copies to symlink if it needs